### PR TITLE
Ensure timestamp is set when sending FedMsg

### LINKED ISSUE #125: sendMessage always has timestamp: 0 on fedmsg
Using jms-messaging-1.1.5 (but there are no changes to src/main/java/com/redhat/jenkins/plugins/ci/messaging/data/FedmsgMessage.java from 1.1.5 to 1.1.7), messages sent via fedmsg always have timestamp: 0 in their messages. It is a little hard to spot, since datagrepper will add its own timestamp. However, it seems just calling ```sendMessage``` prints back the message that is sent out, which includes timestamp: 0. This is a problem because things (e.g. irc notifications) go off this timestamp, and it is resulting in the irc notifications saying XX years ago. When using the activeMQ messages, I don't see a timestamp value auto injected at all (I don't know of any issues with the activeMQ sendMessage). 

For examples, go to this job
https://jenkins-continuous-infra.apps.ci.centos.org/job/fedora-task-pipeline-trigger/
click any of the successful builds console output, and look for
```
Sending message for job 'fedora-task-pipeline-trigger'.
JSON message body:
```
You will see timestamp: 0

Related fedora-ci issue: https://pagure.io/fedora-ci/messages/issue/53

### PR DESCRIPTION
**Pull request by scoheb**
Fixes #125

### CODE REVIEW

File: src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java [hunk 1]
```diff
@@ -269,6 +269,9 @@ public SendResult sendMessage(Run<?, ?> build, TaskListener listener, ProviderDa
             FedmsgMessage fm = new FedmsgMessage(PluginUtils.getSubstitutedValue(getTopic(provider), build.getEnvironment(listener)),
                                                  PluginUtils.getSubstitutedValue(pd.getMessageContent(), env));
 
+            long timeStamp = System.currentTimeMillis();
+            fm.setTimestamp(timeStamp);
```
Comment (ggallen) [inline src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java]: Why not just do this on the same line?

File: src/test/java/com/redhat/jenkins/plugins/ci/integration/FedMsgMessagingPluginIntegrationTest.java [hunk 2]
```diff
@@ -409,6 +409,7 @@ public void testPipelineSendMsgReturnMessage() throws Exception {
         job.sandbox.check(true);
         job.save();
         job.startBuild().shouldSucceed();
+        assertThat(job.getLastBuild().getConsole(), containsString("\"timestamp\":1"));
```
Comment (ggallen) [inline src/test/java/com/redhat/jenkins/plugins/ci/integration/FedMsgMessagingPluginIntegrationTest.java]: Are you sure the timestamp will begin with 1 all the time?

File: src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java (hunk 1 above)
Comment (scoheb) [reply src/main/java/com/redhat/jenkins/plugins/ci/messaging/FedMsgMessagingWorker.java]: fixed

File: src/test/java/com/redhat/jenkins/plugins/ci/integration/FedMsgMessagingPluginIntegrationTest.java (hunk 2 above)
Comment (scoheb) [reply src/test/java/com/redhat/jenkins/plugins/ci/integration/FedMsgMessagingPluginIntegrationTest.java]: Fixed

**Pull request merged**